README.md

```diff
@@ -81,9 +81,26 @@ See the [contributing page](http://jsonnet.org/contributing.html) on our website
 
 ### Running tests
 
-TODO
+```
+make && make libjsonnet.so &&  ./tests.sh
+```
 
 
 ### Locally serving the website
 
-TODO
+First, you need to build the JavaScript version of Jsonnet (which requires [_emscripten_](https://emscripten.org/)):
+```
+make doc/js/libjsonnet.js
+```
+
+Then, inside the _doc_ directory, you can generate and serve the website using [Jekyll](https://jekyllrb.com/):
```

Comment:
There is a script `tools/scripts/serve_docs.sh` which does this but without --livereload.  Perhaps it should use --livereload though.  It must be run from the root of the repo (but it has friendly checks for that).